spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/Interleaved.java

```diff
@@ -34,4 +34,12 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Interleaved {
 
+	/**
+	 * Controls whether the referenced entity should be loaded lazily. This defaults to
+	 * {@literal false}.
+	 *
+	 * @return whether the interleaved property is retrieved lazily.
+	 */
+	boolean lazy() default false;
```

Comment:
I would call the property `fetch` and the value be an enum with the option either Lazy or Eager.

Reply:
Spring Data MongoDB actually uses a boolean named "lazy()" for I think the same reason (that the dependency for javax.persistence.fetchtype isn't a dependency)